### ISSUE TITLE
Question Java

### ISSUE THREAD

**Issue by anneso-duchene** [reddit/IndicatorFastScroll#27]
Hello guys,

My project is old so it's use Java.
I'm trying to use your libray but I have some issue

![issue](https://user-images.githubusercontent.com/2361000/67581908-1a741800-f749-11e9-9a30-46df1bb83488.png)

It's seem that I cannot initialize the item.
Can anyone had this issue ? 

I have try an another way but I still have issue

![Issue2](https://user-images.githubusercontent.com/2361000/67582130-7343b080-f749-11e9-826b-6fc628ee6acb.png)

Thanks in advance : )

**Comment by DSteve595** [comment]
I think you have two options:
1. Enable Java 8 desugaring by adding this to your build.gradle:
```groovy
android {
  compileOptions {
    sourceCompatibility JavaVersion.VERSION_1_8
    targetCompatibility JavaVersion.VERSION_1_8
  }
}
```
This will let you use Java 8 language features, like lambdas and method references.

2. Instantiate an anonymous inner class (`new Function1() { ... }` instead of a lambda.

I'd recommend option 1.

**Comment by anneso-duchene** [comment]
Steve,

I have already enable Java 8. All my lambdas functions are working like this one

![image](https://user-images.githubusercontent.com/2361000/67586197-6034de80-f751-11e9-9e5b-084874778395.png)

For the option 2, I have tried to instiate anonymous inner class by I annot found wich one to use. I have this error for example:

![image](https://user-images.githubusercontent.com/2361000/67586320-a2f6b680-f751-11e9-8985-b565c826a6ea.png)

Thanks for the tips anyway : )

Somebody else ?

**Comment by DSteve595** [comment]
Will try on a Java project and see what's up.

**Comment by anneso-duchene** [comment]
Maybe I need to enable Kotlin library for my project ?

**Comment by DSteve595** [comment]
Are you able to use Kotlin types like `Function1`?

**Comment by anneso-duchene** [comment]
I don't understand : (
Use it like a varibale ?

**Comment by DSteve595** [comment]
If that class can't be imported, it's likely because our library isn't sharing the dependency. Sorry about that. I'll check and fix that.
In the meantime, adding the Kotlin standard library as a dependency will temporarily fix it.

**Comment by anneso-duchene** [comment]
Yeahhhh : D

After adding Kotlin in my Project it's working !!

![image](https://user-images.githubusercontent.com/2361000/67588740-2535a980-f757-11e9-860d-6e3115b72b85.png)

Thanks a lot for your help. It's really a perfect library ! I was searching for it for a long time : )

**Comment by DSteve595** [comment]
Very glad to hear it! Will fix the dependency issue in an update.

**Comment by DSteve595** [comment]
This should be fixed in version 1.2.1. Let me know if you still see any issues.
